src/renderers/shared/stack/event/ReactSyntheticEventType.js

```diff
@@ -8,14 +8,29 @@
  *
  * Flow type for SyntheticEvent class that includes private properties
  *
- * @providesModule ReactSyntheticEvent
+ * @providesModule ReactSyntheticEventType
  * @flow
  */
 
 'use strict';
 
-export type DispatchConfig = any;
+import type {ReactInstance} from 'ReactInstanceType';
+
+export type DispatchConfig = {
+  dependencies: Array<string>,
+  phasedRegistrationNames?: {
```

Comment:
why is this nullable? What's the point of having a dispatch config if there aren't any events that can trigger it?